packages/lib/passport-interface/src/RequestTypes.ts

```diff
@@ -572,7 +572,7 @@ export interface PollFeedResponseValue {
 export interface ZupassUserJson {
   uuid: string;
   commitment: string;
-  email: string;
+  emails: string[];
```

Comment:
This is an incompatible change which will cause open tabs to do something unpredictable when they talk to an updated server.
It might be worth considering changing an API version (embedded in a request URL) so that existing tabs get errors when they connect, rather than downloading state they can't understand.  The route which fetches the latest `self` object would be a good candidate.
I recall using this approach to lock out old code when we improved the sync merging protocol.  This isn't as high-stakes as that, but it is a compatibility-breaking change.

Reply:
done:
- https://github.com/proofcarryingdata/zupass/blob/ivan/account-model/packages/lib/passport-interface/src/api/requestUser.ts#L16
- <img width="709" alt="Screenshot 2024-08-14 at 9 07 55 PM" src="https://github.com/user-attachments/assets/ea895098-3d5d-4387-ba32-1d388689df75">